web-ui/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "arrowParens": "avoid",
+  "bracketSpacing": false,
```

Comment:
May be just a me thing, I sometimes find value in the extra space for imports to increase readability.